Sources/FCM/Helpers/FCM+AccessToken.swift

```diff
@@ -19,9 +19,14 @@ extension FCM {
         .validate()
         .flatMapThrowing { res -> String in
             struct Result: Codable {
-                var access_token: String
+                let access_token: String
             }
-            let result = try res.content.decode(Result.self)
```

Comment:
```swift
guard let body = res.body else {
    throw Abort(.notFound, reason: "Data not found")
}
let result = try JSONDecoder().decode(Result.self, from: res.body)
```
Could you please try this way? NIO have an extension for `JSONEncoder` with byteBuffer
<img width="568" alt="Screenshot 2020-09-15 at 19 47 17" src="https://user-images.githubusercontent.com/1272610/93233568-4696ee00-f78c-11ea-9e44-9fb33baf79f6.png">

Reply:
Or even this way would be better 🙂 
```swift
let result = try res.content.decode(Result.self, using: JSONDecoder())
```

Reply:
yep, right